### DISCUSSION TITLE
Unable to clone this repository on Windows due to invalid filename

### DISCUSSION THREAD

**Discussion by kevinkunker** [triplea-game/triplea#11413] [General]
I don't know if this is the right place to post this, but I'm unable to clone this repository because there is an invalid filename for Windows at C:\triplea-master\game-app\smoke-testing\src\test\resources\junit-platform.properties. It looks like there is a character at the end of junit-platform.properties that sometimes appears as a blank space. Whatever it is, it would not let me clone the repo until I deleted the file in my fork.

**Comment by RoiEXLab** [comment]
This is definitely the right place to post this (maybe an issue would've been better, but doesn't really matter).
@DanVanAtta Looks like it also breaks on my machine when I try to update my branch. Can you have a look into this?

**Reply by RoiEXLab** [reply]
@kevinkunker A bit late, but it should work again: #11511
